index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/style.css" />
+    <title>JS Conditional Code</title>
+  </head>
+  <body>
+    <nav id="navbar">
```

Comment:
Please note that classes are better to work with CSS, ids should be used only where necessary.
Read more about why classes are better than ids [here](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4).